src/Compilers/CSharp/Portable/BoundTree/BoundNodeExtensions.cs

```diff
@@ -94,5 +97,152 @@ private class ContainsAwaitVisitor : BoundTreeWalkerWithStackGuardWithoutRecursi
                 return null;
             }
         }
+
+        /// <summary>
+        /// Visits the binary operator tree of interpolated string additions in a depth-first in-order visit.
```

Comment:
>depth-first in-order [](http://example.com/codeflow?start=82&length=20)

I think it would be clearer to specify the order explicitly. There are three nodes (left, right, parent), in what order are they supposed to be visited?